### PR TITLE
Ticket #16 Edit a tag

### PR DESCRIPTION
**Pull request by gabe-grant**
# Description

As an admin I would like to be able to modify a Tag so that I can rephrase the name if I think of something more appropriate.

## Type of change

New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Tested by adding a NEW tag, redirected to the tag list I selected the tag I created and changed the name of the property and clicked **Save**. I was in fact redirected back to the tag list and saw the change updated in the database. 

Tested also by updated a different tag by selecting the **Edit** link beside the tag and was in fact directed to the edit form. Implemented a change to the tag name and instead selected the **Back to list** link and was successfully directed back to the webpage that displayed all the correct list of tags. 

Try it for yourselves in the exact order above.

**Test Configuration**:
* Firmware version: Windows 10 Pro
* Hardware: DELL Latitude 5590
* Toolchain: 3.1.5 Visual Studio
* SDK: Nuget Microsoft.Data.SqlClient

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: TabloidMVC/Views/Tags/Edit.cshtml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            </div>
+            <div class="form-group">
+                <input type="submit" value="Save" class="btn btn-primary" />
+            </div>
+        </form>
+    </div>
+</div>
+
+<div>
+    <a asp-action="Index">Back to List</a>
```
Comment (jordan-castelloe) [inline TabloidMVC/Views/Tags/Edit.cshtml]: According to the ticket, this should be a cancel button

**Review by jordan-castelloe — CHANGES_REQUESTED** [review]
Great job, Gabe! Works as expected. I made one small comment-- according to the ticket the "Back to List" link should be changed to a cancel button. Minor change! Once you push up that fix, @ me on slack and I'll give this my official stamp of approval

**Comment by jordan-castelloe** [comment]
Also, great PR testing instructions 👍

**Comment by gabe-grant** [comment]
I fixed that jordan!

**Review by jordan-castelloe — APPROVED** [review]
Looks great!! Nice work!

**Pull request merged**